components/dashboard/src/components/QuickStart.tsx

```diff
@@ -87,6 +119,18 @@ const QuickStart: FC = () => {
             const searchParams = new URLSearchParams(window.location.search);
             searchParams.delete("message");
 
+            if (hashValue.startsWith("quickstart-")) {
+                const storedHash = localStorage.getItem(hashValue);
```

Comment:
We optimistically assume that the availability of `localStorage` does not change between the time you initiate the quickstart to the point where you return from auth.